accelerator/core/serializer/serializer.c

```diff
@@ -694,7 +717,192 @@ status_t ta_send_transfer_res_serialize(ta_send_transfer_res_t* res, char** obj)
   return ret;
 }
 
-status_t receive_mam_message_res_serialize(char* const message, char** obj) {
+static status_t recv_mam_message_mam_v1_req_deserialize(cJSON const* const json_obj, ta_recv_mam_req_t* const req) {
+  cJSON *json_sub = NULL, *json_value = NULL;
+  status_t ret = SC_OK;
+  char *bundle_hash = NULL, *chid = NULL, *epid = NULL, *msg_id = NULL, *psk = NULL, *ntru = NULL;
```

Comment:
Make the naming of `chid`, `edid` and `meg_id` consistent.

What is `psk` and `ntru` indicated?

Reply:
The naming of `chid`, `epid`, `msg_id` follows the naming in `entangled`.
`psk` and `ntru` are the descryption cryptograpghy used in MAM